### DISCUSSION TITLE
If i want to change input size from data loader then what are the necessary changes to be made in config file

### DISCUSSION THREAD

**Discussion by shivam228** [bowenc0221/panoptic-deeplab#108] [Q&A]
I am trying to downsample input so that i can train model faster.but I am bit confused how to proceed.

**Comment by bowenc0221** [comment]
https://github.com/facebookresearch/detectron2/blob/c081505af16f54c5a013916b81636d03f7c0a8fd/projects/Panoptic-DeepLab/configs/Cityscapes-PanopticSegmentation/Base-PanopticDeepLab-OS16.yaml#L53-L62
